### PR TITLE
Primate AI 3D

### PR DESCRIPTION
**Pull request by Fahd-Siddiqui**
Primate AI 3D

Jira Link: https://nirvana-annotator.atlassian.net/browse/NIR-1519 

added documentation for Primate AI 3D

### CODE REVIEW

File: docs/data-sources/primate-ai.mdx [hunk 1]
```diff
@@ -6,24 +6,124 @@ import JSON from './primate-ai-json.md'
...
+* `alt_aa`
+* `score_PAI3D`
+* `percentile_PAI3D`
+* `refseq`
+
+The fields `gene_name` and `refseq` define the Ensembl and RefSeq transcript IDs respectively.
+These transcripts are passed as-is and some of them might be unrecognized/deprecated by RefSeq/Ensembl.
+
+#### Parsing Run
+```text
```
Comment (rajatshuvro) [inline docs/data-sources/primate-ai.mdx:58]: Not sure if having the parsing run output is useful, but having the command too would benefit the users. So, let's add the command too.

File: docs/data-sources/primate-ai.mdx [hunk 2]
```diff
@@ -6,24 +6,124 @@ import JSON from './primate-ai-json.md'
...
+  "scoreMaximum": 0.9988
+}
+
+Time: 00:42:40.7
+```
+
+
+### Known Issues
+
+:::caution Known Issues
```
Comment (rajatshuvro) [inline docs/data-sources/primate-ai.mdx:101]: excellent.

**Review by rajatshuvro — CHANGES_REQUESTED** [review]
Let's add the command that generated primate-ai-3d.

**Comment by Fahd-Siddiqui** [comment]
Primate AI 3D documentation is added in #36

**Pull request closed without merging**